<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProductionPlanningStory</class>
 <widget class="QWidget" name="ProductionPlanningStory">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>968</width>
    <height>861</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QLabel" name="text">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>901</width>
     <height>531</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 26pt &quot;Segoe Script&quot;;
color: rgb(0, 0, 127);</string>
   </property>
   <property name="text">
    <string>Félix  visita la coopérative de fermiers voisins. 
 Les fermiers étaient débordés de récoltes
  et ne savaient pas comment organiser 
 leur travail pour être le plus efficace possible. 
 
  Félix, avec son esprit vif, 
 suggéra une méthode pour organiser les tâches 
 agricoles de manière à ce que tout 
 se déroule comme sur des roulettes.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="toProductionPlanningPage">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>620</y>
     <width>311</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(250, 250, 186);
border-color: rgb(170, 170, 127);
font: 12pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>voir la solution de Félix</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>560</y>
     <width>471</width>
     <height>321</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>producionPlannig.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>text</zorder>
  <zorder>label</zorder>
  <zorder>toProductionPlanningPage</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
